lib/index.js

```diff
@@ -137,65 +137,91 @@ module.exports = babel => {
 			Program: {
 				enter() {
 					this.moduleConfigs = normalizeModulesConfig(this.opts.modules);
+					this.failOnError = typeof this.opts.failOnError === 'boolean' ? this.opts.failOnError : true;
```

Comment:
Generally I prefer the following for options which are true unless explicitly false:
```js
this.failOnError = this.opts.failOnError !== false;
this.logOnError = this.opts.logOnError !== false;
```